tests/json_common_interface_test.rb

```diff
@@ -123,4 +123,43 @@ def test_JSON
     assert_equal @json, JSON(@hash)
     assert_equal @hash, JSON(@json)
   end
+
+
+  def test_load_file
```

Comment:
Please add a test with options...

Reply:
Sorry about that. I've added this (and an equivalent function for `load_file!`), would it suffice?:

```
  def test_load_file_with_option
    temp_file_containing(@json) do |filespec|
      key_classes = load_file(filespec, symbolize_names: true).keys.map(&:class)
      assert_true (key_classes.include?(Symbol) && (! key_classes.include?(String)))
    end
  end
```

Also, in order to get the tests to pass I had to `rebase master` my feature branch. Is it ok to `git push -f` this branch?

Reply:
> Is it ok to git push -f this branch?

Yes, that's the best thing to do 😄